### DISCUSSION TITLE
Roborock S5: Segments not created after cleanup

### DISCUSSION THREAD

**Discussion by jurispucekts** [Hypfer/Valetudo#1896] [Q&A / Support] [answered]
Facing this problem while trying to run multiple resets/cleanups. Never are segments created (just one segment). Valetudo version 2023.10, Robrock S5. Multiple rooms in there with doors, etc. Tried disabling/enabling persistent maps, etc. I have seen others struggling with this but no response I have seen has worked for me. "Triggering" recreation through "quirks" doesn't help

**Comment by Hypfer** [comment, marked as answer]
The Roborock S5 is a bit special in that regard as it received segment support as an update during its lifecycle.

Make sure that you're running at least fw 2008 (recommended would be 2034) as earlier firmware versions can't do segments.

**Reply by jurispucekts** [reply]
Thanks - will check when I am at home. Somehow have missed this response in forums I have googled ;)
